packages/components/src/components/progress-bar-onboarding/progress-bar-onboarding.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import './progress-bar-onboarding.scss';
+
+type TProgressBarOnboarding = {
+    step: number;
+    amount_of_steps: Array<string>;
```

Comment:
can't we pass a number of steps instead of array of string since we just using the length of this array?

Reply:
In this component we mostly get an array of our keys which are taken from contents in our onboarding component. That is why I am getting the amount of steps from and array. I can also add number as one of the types so that we can have a wider range of types which are supported in the component.